addon/templates/components/materialize-collection-item.hbs

```diff
@@ -0,0 +1,13 @@
+{{#if isHeader}}
+  <h4>{{yield}}</h4>
+{{else}}
+  {{#if image}}
+    <img class="circle" alt="{{imageAlt}}" src="{{image}}"/>
+  {{/if}}
+  {{#if title}}
+    <span class="title">{{title}}</span>
+    <p>{{yield}}</p>
+  {{else}}
+    {{yield}}
+  {{/if}}
+{{/if}}
```

Comment:
Do you have maybe an idea how to implement it more efficient? I agree with you that it should be as lightweight as possible, but this was my best solution to implement every single case of how the content should be displayed